app/workers/emails/enqueue_digest_worker.rb

```diff
@@ -1,4 +1,4 @@
-module Email
+module Emails
```

Comment:
IIRC when the name of a worker is changed we should add an alias for a while, right?

Reply:
Nah, this will be fine and I actually want this right now bc I need to kill the job running.